File: src/app/shared/model/disco.ts
export class Disco {

    constructor(public titulo?: string, public barCode?: string, public banda?: string, public genero?: string, public gravadora?: string) {
        this.titulo = titulo;
        this.barCode = barCode;
        this.banda = banda;
        this.genero = genero;
        this.gravadora = gravadora;
    }
    public getTitulo(){
        return this.titulo;
    }
    public getBarCode(){
        return this.barCode;
    }
    public getBanda(){
        return this.banda;
    }
    public getGenero(){
        return this.genero;
    }
    public getGravadora(){
        return this.gravadora;

    }
    public setTitulo(titulo: string){
         this.titulo = titulo;
    }
    public setBarCode(barCode: string){
         this.barCode = barCode;
    }
    public setBanda(banda: string){
         this.banda = banda;
    }
    public setGenero(genero: string){
        this.genero = genero;
    }
    public setGravadora(gravadora: string){
        this.gravadora = gravadora;

    }
    toString(): string{
        return `Título: ${this.titulo}
       / Banda: ${this.banda}
       / Gênero: ${this.genero}
       / Gravadora: ${this.gravadora} `;
    }
}

File: api/api/src/main/java/br/edu/ifpb/vinyljoy/vinyljoy/model/Disco.java
package br.edu.ifpb.vinyljoy.vinyljoy.model;


import jakarta.persistence.Id;

import jakarta.persistence.Table;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;

@Entity
@Table(name = "tb_disco")
public class Disco {

   @Id
   @GeneratedValue(strategy = GenerationType.AUTO)
   private String barCode;
   private String titulo;
   private String banda;
   private String genero;
   private String gravadora;
 

 
   public String getTitulo() {
       return titulo;
   }

   public void setTitulo(String titulo) {
       this.titulo = titulo;
   }

   public String getBarCode() {
       return barCode;
   }

   public void setBarCode(String barCode) {
       this.barCode = barCode;
   }

   public String getBanda() {
       return banda;
   }

   public void setBanda(String banda) {
       this.banda = banda;
   }
   
   public String getGenero() {
       return genero;
   }

   public void setGenero(String genero) {
       this.genero = genero;
   }
   public String getGravadora() {
       return gravadora;
   }

   public void setGravadora(String gravadora) {
       this.gravadora= gravadora;
   }
}
